client/my-sites/comments/comment-list/index.jsx

```diff
@@ -55,12 +55,17 @@ export class CommentList extends Component {
 
 	state = {
 		isBulkEdit: false,
+		persistedComments: [],
+		// TODO: replace {} with [] after persistedComments is merged
```

Comment:
Basically the change from "list of comments" to "list of comment IDs" in this PR breaks the current selection implementation.
It needs to be slightly adapted and, on top of the additional code to review, it would have needed more testing efforts, since the selection is currently disabled via feature flag.